### PR TITLE
Fix CS using Spryker+Slevomat cs ruleset.

### PR DESCRIPTION
**Pull request by dereuromark**
This is the first run of the sniffer for cs.

Please double check that it didn't modify code wrongly.

### CODE REVIEW

File: src/Propel/Runtime/Formatter/ArrayFormatter.php [hunk 1]
```diff
@@ -46,7 +59,7 @@ public function format(DataFetcherInterface $dataFetcher = null)
         foreach ($dataFetcher as $row) {
             $object = &$this->getStructuredArrayFromRow($row);
             if ($object) {
-                $items[] =& $object;
+                $items[] = & $object;
```
Comment (gechetspr) [inline src/Propel/Runtime/Formatter/ArrayFormatter.php:62]: isn't it should be together?

Comment (dereuromark) [reply src/Propel/Runtime/Formatter/ArrayFormatter.php:62]: From techn. perspective this doesnt matter. But code style wise I think it should belong to the var even (without space).
A follow up CS fixer update IMO.

**Comment by dereuromark** [comment]
I didnt change any functional parts (except for some casts that are obvious)

Errors left are now (and require follow up PR):
```
FILE: /Propel2/src/Propel/Generator/Builder/Util/PropelTemplate.php
---------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------------
 96 | WARNING | Use of eval() is discouraged (Squiz.PHP.Eval.Discouraged)
---------------------------------------------------------------------------------------------------------------------------


FILE: /Propel2/src/Propel/Generator/Builder/Om/ObjectBuilder.php
---------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 6 ERRORS AFFECTING 6 LINES
---------------------------------------------------------------------------------------------------------------------------------------------------
 3771 | ERROR | Method name "ObjectBuilder::addGetPrimaryKey_SinglePK" is not in camel caps format
      |       | (PSR1.Methods.CamelCapsMethodName.NotCamelCaps)
 3798 | ERROR | Method name "ObjectBuilder::addGetPrimaryKey_MultiPK" is not in camel caps format
      |       | (PSR1.Methods.CamelCapsMethodName.NotCamelCaps)
 3836 | ERROR | Method name "ObjectBuilder::addGetPrimaryKey_NoPK" is not in camel caps format (PSR1.Methods.CamelCapsMethodName.NotCamelCaps)
 3878 | ERROR | Method name "ObjectBuilder::addSetPrimaryKey_SinglePK" is not in camel caps format
      |       | (PSR1.Methods.CamelCapsMethodName.NotCamelCaps)
 3908 | ERROR | Method name "ObjectBuilder::addSetPrimaryKey_MultiPK" is not in camel caps format
      |       | (PSR1.Methods.CamelCapsMethodName.NotCamelCaps)
 3942 | ERROR | Method name "ObjectBuilder::addSetPrimaryKey_NoPK" is not in camel caps format (PSR1.Methods.CamelCapsMethodName.NotCamelCaps)
---------------------------------------------------------------------------------------------------------------------------------------------------


FILE: /Propel2/src/Propel/Generator/Builder/Om/TableMapBuilder.php
----------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
----------------------------------------------------------------------------------------------------------------------------------------------------------------
  990 | ERROR | Method name "TableMapBuilder::addGetOMClass_Inheritance" is not in camel caps format (PSR1.Methods.CamelCapsMethodName.NotCamelCaps)
 1057 | ERROR | Method name "TableMapBuilder::addGetOMClass_NoInheritance" is not in camel caps format (PSR1.Methods.CamelCapsMethodName.NotCamelCaps)
 1085 | ERROR | Method name "TableMapBuilder::addGetOMClass_NoInheritance_Abstract" is not in camel caps format
      |       | (PSR1.Methods.CamelCapsMethodName.NotCamelCaps)
----------------------------------------------------------------------------------------------------------------------------------------------------------------


FILE: /Propel2/src/Propel/Generator/Manager/AbstractManager.php
-----------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------------------
 508 | WARNING | Silencing errors is discouraged; found: @file($file... (Generic.PHP.NoSilencedErrors.Discouraged)
-----------------------------------------------------------------------------------------------------------------------
```

**Comment by dereuromark** [comment]
PHP Stan level 7 is down to 252 errors left (from 1500 originally).

File: src/Propel/Generator/Builder/Om/ObjectBuilder.php [hunk 2]
```diff
@@ -5834,11 +5837,13 @@ public function set{$relatedNamePlural}(Collection \${$inputCollection}, Connect
     /**
      * @param string $script
      * @param \Propel\Generator\Model\CrossForeignKeys $crossFKs
+     *
+     * @return void
      */
     protected function addCrossFKCount(&$script, CrossForeignKeys $crossFKs)
     {
         $refFK = $crossFKs->getIncomingForeignKey();
-        $selfRelationName = $this->getFKPhpNameAffix($refFK, $plural = false);
```
Comment (gechetspr) [inline src/Propel/Generator/Builder/Om/ObjectBuilder.php:5532]: We have this issue in 44 places in this file currently

Comment (dereuromark) [reply src/Propel/Generator/Builder/Om/ObjectBuilder.php:5532]: I fixed those now, too.

**Comment by dereuromark** [comment]
phpstan level 6 now in travis

(phpstan level 7 has 244 errors left)

**Pull request merged**